README.md

```diff
@@ -281,6 +281,38 @@ Note that at this point in time, Regal only considers the line following the ign
 entire blocks of code (like rules, functions or even packages). See [configuration](#configuration) if you want to
 ignore certain rules altogether.
 
+## Using Regal with Pre-Commit
+
+Add this to your `.pre-commit-config.yaml`
```

Comment:
For people unfamiliar with the pre-commit project, it'd be good to add a link here.

Reply:
Updated with link and minor verbiage, wdyt?